src/index.js

```diff
@@ -1,10 +1,10 @@
 /* jshint expr:true */
 import Amplitude from  './amplitude';
 
-var old = window.amplitude || {};
-var newInstance = new Amplitude();
+const old = window.amplitude || {};
+const newInstance = new Amplitude();
```

Comment:
Alternatively maybe we ought to just add `window.amplitude = amplitude` right here for the build that the snippet will use..